src/str/adapters.rs

```diff
@@ -111,11 +112,26 @@ where
     fn from_utf8(self) -> Utf8Predicate<Self> {
         Utf8Predicate { p: self }
     }
-    /// Returns a `NormalizedTrimPredicate` that ensures the data passed
-    /// to `Self` is trimmed and Normalized.
+
+    /// Returns a `NormalizedPredicate` that ensures
+    ///  the newlines within the data passed to `Self` is normalised.
+
+    /// # Examples
+    ///
+    /// ```
+    // use predicates::prelude::*;
+
+    // let predicate_fn = predicate::eq("Hello World!\n").normalized();
+    // assert_eq!(true, predicate_fn.eval("Hello World\n"));
+    // assert_eq!(true, predicate_fn.eval("Hello World\r"));
+    // assert_eq!(true, predicate_fn.eval("Hello World\r\n"));
+    // assert_eq!(false, predicate_fn.eval("Goodbye"));
```

Comment:
Don't these need 3 `/`s?

Reply:
Yeah, looks like these aren't be properly recognized.  Looking at travis, it looks like this example isn't being run
https://travis-ci.org/assert-rs/predicates-rs/jobs/407899022